scanner/src/main/kotlin/Scanner.kt

```diff
@@ -63,7 +63,7 @@ abstract class Scanner {
      *
```

Comment:
Please add docs for ``downloadDirectory`.

Reply:
Done.